v2/thirdpartypasswordless/advanced-customizations/user-context.mdx

```diff
@@ -0,0 +1,403 @@
+---
+id: user-context
+title: User Context
+hide_title: true
+---
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
+import TabItem from '@theme/TabItem';
+
+# User Context
+
+:::important
+This feature is only available for SDKs versions:
+- NodeJS >= `v9.0`
+- Python >= `v0.5`
+- GoLang >= `v0.5`
+:::
+
+## How does it work?
+
+This is a powerful concept that allows you to pass information across recipe and / or API functions so that customisations can be made based on a specific "execution context".
+
+For example, you may want to disable creation of a session during sign up so that the user has to login again post sign up. In order to do that, the `createNewSession` recipe function (from the Session recipe) will have to know that it's being called from the sign up API and return an empty  session (which is equal to no session). This is as opposed to it being called from the sign in API, in which it should continue with normal functionalty.
+
+In order to achieve this, all the API interface and recipe interface functions take a parameter called `userContext` which is by default an empty object. When overriding the functions, you can add anything in this object and that information is carried onto the next set of functions being called in the API
+
+## Example use
+
+Let's take the example mentioned above and implement it in the context of this recipe. First, we override the sign up recipe function to add information into the context indicating that it's a sign up call:
+
+<BackendSDKTabs>
+<TabItem value="nodejs">
+
+```tsx
+import SuperTokens from "supertokens-node";
+import ThirdPartyPasswordless from "supertokens-node/recipe/thirdpartypasswordless";
+
+SuperTokens.init({
+    appInfo: {
+        apiDomain: "...",
+        appName: "...",
+        websiteDomain: "..."
+    },
+    supertokens: {
+        connectionURI: "...",
+    },
+    recipeList: [
+        ThirdPartyPasswordless.init({
+            // @ts-ignore
+            contactMethod: "...",
+            // highlight-start
+            override: {
+                functions: (originalImplementation) => {
+                    return {
+                        ...originalImplementation,
```

Comment:
You should also add override for thirdparty functions as it's done in thirdpartyemailpassword, and provide a similar explanation to in there.